spec/requests/physical_storages_spec.rb

```diff
@@ -174,4 +174,26 @@
       end
     end
   end
+
+  context 'Physical Storages edit action' do
+    it 'OPTIONS /api/physical_storages/:id' do
```

Comment:
@llivne I don't see an options call on the collection or the instance for physical_storages, nor do I see `params_for_{create,update}` in the provider.  Where are the UI params coming from?

Reply:
@agrare all of the params are in the ddf form inside the ui-classic (app/javascript/components/physical-storage-form/physical-storage-form.schema.js).

we therefore dont have params_for_{create,update} in the provider since the form is generic.

Reply:
Okay what are we checking for in this test then?  Isn't this just the "default" options handler for the collection?

Reply:
@agrare you are correct. I used cloud_volumes editing as a reference but in ph_storage editing we are not using OPTIONS  /api/physical_storages/:id so I removed this test.